src/nmm/grid.js

```diff
@@ -115,10 +115,10 @@ export class Grid {
         return coins;
     }
 
-    checkAvailablePos() {
-        for (let row = 0; row < this.#gridInit.length; row++) {
-            for (let col = 0; col < this.#gridInit.length; col++) {
-                if (this.#gridInit[row][col].symbol === Grid.coinSymbols.PA) {
+    checkAvailablePos(grid) {
```

Comment:
la entrada tiene que ser una posicion no todo el grid, tiene que devolver true en UNA POSICION DETERMIDA, no si existe una posicion vacía en todo el grid.

Reply:
Hi @dkmar11 , lets keep using english for written communication 👍🏽